numpy/ma/core.py

```diff
@@ -3678,6 +3678,16 @@ def __repr__(self):
             return _print_templates['short_std'] % parameters
         return _print_templates['long_std'] % parameters
 
+    def _delegate_binop(self, other):
```

Comment:
This is an improvement, but the fact that it is needed worries me. What other applications are going to break because of this change?